README.md

```diff
@@ -13,31 +13,27 @@ might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
 
-# Revised Answer:
+# Answer:
 
-If we cannot assume a particular T(n), then we must prove that 
+If we cannot assume a particular T(n) or c, then we must prove that 
 
 $O(log_{2} n) \subseteq O(log_{5} n)$ and $O(log_{5} n) \subseteq O(log_{2} n)$
 
-$log_{2} n$ = $\frac {log_{5} n} {log_{5} 2}$
+T(n) $\leq c \cdot log_{5} n$
 
-c = $\frac {1} {log_{5} 2}$
+$log_{5} n$ = $\frac {log_{2} n} {log_{2} 5} $
 
-$log_{2} n \leq c \cdot log_{5} n$
+T(n) $\leq c \cdot \frac {log_{2} n} {log_{2} 5}$
 
-$\frac {log_{5} n} {log_{5} 2} \leq \frac {1} {log_{5} 2} \cdot log_{5} n$ (They are equal)
+T(n) $\leq c \cdot log_{2} n$
```

Comment:
Where did the $\log_2 5$ go here?

Reply:
I combined it with c because it is a constant.

Reply:
Ok, please be explicit about this. Also sounds like this would be a new constant.